proj/sub1/test/b.ml

```diff
@@ -2,13 +2,11 @@
    Tests for Sub1.B
 *)
 
-let test_string () =
-  Alcotest.(check (neg string)) "foo is not bar" "foo" "bar"
+let test_string () = Alcotest.(check (neg string)) "foo is not bar" "foo" "bar"
 
-let test_string_hasty () =
-  assert ("foo" <> "bar")
+let test_string_hasty () = assert ("foo" <> "bar")
 
-let tests = [
-  "string", `Quick, test_string;
-  "string, hasty", `Quick, test_string_hasty;
-]
+let tests =
+  [
+    ("string", `Quick, test_string); ("string, hasty", `Quick, test_string_hasty);
+  ]
```

Comment:
I'm not a fan of this formatting style for lists but I'll get used to it.

Reply:
This was autogenerated by ocamlformat :D